cli.js

```diff
@@ -54,6 +68,30 @@ const formatOptionValidator = (opt) => {
 	return option;
 };
 
+/**
+ * Validator method for the size option flag
+ * @param opt {string} The size option flag value
+ * @returns {string} The processed size option flag value
+ */
+const sizeOptionValidator = (opt) => {
+
+	const allowedSizes = Object.entries(SUPPORTED_SIZES).map(([size, allowed]) => {
+		if (allowed) return size;
+	}).concat('all');
+
+	if (opt === true) {
+		throw new Error(`You have to choose one of these sizes: ${allowedFormats}`);
+	}
+
+	const option = opt.charAt(0).toUpperCase() + opt.slice(1).toLowerCase()
```

Comment:
Why d you choose only the first two characters ?

Reply:
I'm taking the whole string. `opt.slice(1)` is taking the substring that starts from the 2nd character.

Reply:
My bad, I wan't paying attention enough 😳 and also forgot how caporal handles arguments.